<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MPDU_OutputSwWid</class>
 <widget class="QWidget" name="MPDU_OutputSwWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1420</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_13" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>20</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>730</y>
     <width>71</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>全选</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_21" native="true">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>500</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QWidget" name="widget_20" native="true">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>500</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>660</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MPDU开关测试工具</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>100</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QWidget" name="widget_15" native="true">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>340</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QRadioButton" name="openRadio">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>730</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>全开</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_9" native="true">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>180</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QWidget" name="widget_7" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QWidget" name="widget_14" native="true">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>340</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QWidget" name="widget_1" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QWidget" name="widget_22" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QWidget" name="serialWid" native="true">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>0</y>
     <width>255</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>330</y>
     <width>231</width>
     <height>511</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QWidget" name="widget_16" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QWidget" name="widget_6" native="true">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>100</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>730</y>
     <width>140</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_24" native="true">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>590</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QWidget" name="widget_10" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QWidget" name="widget_23" native="true">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>590</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>730</y>
     <width>54</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>地址：</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_8" native="true">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>180</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QWidget" name="widget_18" native="true">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>420</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QWidget" name="widget_17" native="true">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>420</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QWidget" name="widget_12" native="true">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>260</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>730</y>
     <width>71</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QRadioButton" name="closeRadio">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>730</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>全关</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QWidget" name="widget_11" native="true">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>260</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QWidget" name="widget_19" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
